connect_transformations/formula/mixins.py

```diff
@@ -52,7 +52,8 @@ def formula(
                     parameters['additional_parameters'] = {'precision': expression['precision']}
                 result[expression['to']] = cast_value_to_type(**parameters)
             except Exception as e:
-                return RowTransformationResponse.fail(output=str(e))
+                if not expression.get('ignore_errors'):
```

Comment:
Probably we should add to result:

`result[expression['to']] = None`

if errors must be ignored right ?